pkgs/development/tools/misc/ccls/default.nix

```diff
@@ -13,9 +13,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake makeWrapper ];
-  buildInputs = with llvmPackages; [ clang-unwrapped llvm rapidjson ];
+  buildInputs = with llvmPackages; [ clang clang-unwrapped llvm rapidjson ];
```

Comment:
`clang` shouldn't be needed if it's already a clang based stdenv.

Reply:
That's correct. Thanks!